tutorials/prompts/react_prompting.ipynb

```diff
@@ -50,7 +50,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # from phoenix.client.experiments import run_experiment
</pre><p>don't you need these?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/9291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>